NEWS.md

```diff
@@ -1,9 +1,13 @@
 # ggplot2 3.0.0.9000
 
-* `geom_hex()` now understands the `size` and `linetype` aesthetics
-  (@mikmart, #2488).
-  
-# ggplot2 3.0.0
+## Minor bug fixes and improvements
```

Comment:
Can you remove this heading please? It's easier to add these just before release. (http://style.tidyverse.org/news.html#news-bullets)

Reply:
Good you flagged that. I had also messed up the main heading in the rebase. Should be good now.